src/Sylius/Bundle/ApiBundle/Validator/Constraints/PromotionCouponEligibility.php

```diff
@@ -17,8 +17,11 @@
 
 final class PromotionCouponEligibility extends Constraint
 {
-    /** @var string */
-    public $message = 'sylius.promotion_coupon.is_invalid';
+    public string $invalid = 'sylius.promotion_coupon.is_invalid';
```

Comment:
Probably not acceptable by BC promise anymore.

```suggestion
    public string $invalid = 'sylius.promotion_coupon.invalid';
```